en/docs/references/postman-integration.md

```diff
@@ -0,0 +1,129 @@
+# Test Your Service via Postman
+
+The Postman Editor allows you to integrate with your Postman account to easily test your services.
+
+## Prerequisites
+
+In order to use the Postman Editor:
+
+- Make sure you have logged into the Choreo console and created a service. For instructions to create a service, see [Create Your First Service](../services/create-your-first-service.md).
+
+    !!! tip
+        This guide uses the service created in [Create Your First Service](../services/create-your-first-service.md) as an example.
+        
+- Have an active Postman account.
+
+## Step 1: Generate Postman API key
+
+To generate the Postman API key so that you can connect your Postman account to Choreo, follow this procedure:
+
+1. Sign in to your Postman account.
+
+2. Go to your profile, and click **Account Settings**. Then click **Postman API Keys**
+
+    ![Generate Postman API key](../assets/img/postman/generate-postman-api-key.png){.cInlineImage-full}
+
+3. Click **Generate API Key**.
+
+4. In the **Generate an API key** dialog box, enter `choreo` as the key name. Then click **Generate API Key**.
```

Comment:
It is not mandatory to name it as `choreo`. So shouldn't we reword this to say enter the key name , e.g. `choreo` or something similar to that?